index.html

```diff
@@ -127,19 +116,21 @@ <h2>JOIN US!</h2>
 
                 <div class="class-section">
 
-                    <h2>Our Classes</h2>
-                    <p>We have hosted classes on Python, Javascript, HTML/CSS, Arduino, data analytics, and more.</p>
-                    <br>
-                    <p>To view the recordings or presentation slides of our previous virtual meetings, please click on the following
-                        <a href="https://docs.google.com/document/d/1iBMCb9s0rgvN9bEptp1l83BMtzx2CWVme-IPx60E-6I/edit?usp=sharing">document link</a>.</p>
+                    <h2>Our Classes and Events</h2>
```

Comment:
I really like the inclusion of Events alongside Classes. I'd like to see us try a couple more of those this year.